utils/checkExpiry.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Check if date is expired (defining expired as 1 year from the date)
+ * @param date the date we to check
+ * @returns true if expired, false if not
+ */
+export default function checkExpiry(date: Date): boolean {
+    const d = new Date(date);
+    const expiryDate = new Date(d.getUTCFullYear() + 1, d.getMonth(), d.getDate());
```

Comment:
Any reason why you are mixing UTC year with client month/dates? Sine this is only used client side, maybe do:
```suggestion
    const expiryDate = new Date(d.getFullYear() + 1, d.getMonth(), d.getDate());
```